webapp/src/types/index.d.ts

```diff
@@ -8,7 +8,7 @@ export interface IAppWindow extends Window {
     msCrypto: Crypto
     openInNewBrowser?: ((href: string) => void) | null
     webkit?: {messageHandlers: {nativeApp?: {postMessage: <T>(message: T) => void}}}
-    openPricingModal?: () => () => void
+    openPricingModal?: () => (callerInfo: string) => void
```

Comment:
Nit: use a name more specific to purpose, maybe something like `trackingLocation` ?